src/org/omegat/util/Log.java

```diff
@@ -324,4 +323,65 @@ public static void logDebug(Logger logger, String message, Object... parameters)
             logger.log(rec);
         }
     }
+
+    /**
+     * Writes a warning message through a specified logger.
+     * 
+     * @param logger
+     *            logger object
+     * @param key
+     *            The key of the error message in the resource bundle
+     * @param parameters
+     *            Parameters for the error message.
+     */
+    public static void logWarningRB(Logger logger, String key, Object... parameters) {
```

Comment:
Are there any good ways to  abstract a logger without referencing `java.util.logging.Logger` ?